src/humanize/i18n.py

```diff
@@ -5,7 +5,7 @@
 import os.path
 from threading import local
 
-__all__ = ["activate", "deactivate", "thousands_separator"]
+__all__ = ["activate", "deactivate", "thousands_separator", "decimal_separator"]
```

Comment:
Alphabetical here too:
```suggestion
__all__ = ["activate", "deactivate", "decimal_separator", "thousands_separator"]
```